src/delagent/ui/admin-upload-delete.php

```diff
@@ -190,6 +193,14 @@ public function Output()
     $text = _("Delete");
     $V.= "<input type='submit' value='$text'>\n";
     $V.= "</form>\n";
+
+    $V.= <<<EOT
```

Comment:
I choose multiline string syntax, if I need to use a PHP variable e.g. here:
<pre>phpversion($currentExtensionName); </pre>
https://github.com/siemens/fossology/blob/0159a0f38590e5d8739e401619da039b9b9d6102/src/www/ui/admin-dashboard.php#L129-L165

Otherwise I use EOT, because it does look better imho and you can be sure that there is no PHP code executed in the EOT.

Is that bad practice?